WordPress/Classes/ViewRelated/Views/List/NotificationsList/NotificationsTableViewCellContent.swift

```diff
@@ -37,10 +37,12 @@ struct NotificationsTableViewCellContent: View {
         switch style {
         case .regular(let regular):
             Regular(info: regular)
-                .padding(.bottom, Length.Padding.medium)
+                .padding(.top, Length.Padding.split)
```

Comment:
Nit: we can combine these 2 lines: `.padding(.horizontal, Length.Padding.split)`

The same goes for the one below.